src/index.js

```diff
@@ -0,0 +1,157 @@
+
+const state = {
+    tempDeg: 0,
+    defaultColor: "teal",
+    cityName : "Seattle",
+    defaultLandscape: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+    hotLandscape: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+    niceLandscape: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+    midLandscape: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+    increaseTempButton: null,
+    decreaseTempButton: null,
+    tempValue: null, 
+    landscape: null,
+    cityNameOutput : null,
+    cityNameInput: null,
+    currentWeatherButton : null,
+    resetButton : null,
+    skySelect : null,
+    sky : null,
+};
+
+const setDefaultValues = () => {
+    state.cityNameInput.value = `${state.cityName}`;
+    state.cityNameOutput.textContent = `${state.cityName}`;
+    state.landscape.textContent = `${state.defaultLandscape}`;
+    tempValue.style.color = state.defaultColor;
+    tempValue.textContent = 0;
+    sky.textContent = "";
+};
+
+const registerEventHandlers = () => {
+    state.increaseTempButton.addEventListener("click", increaseTemp, changeColor, changeLandscape);
+    state.decreaseTempButton.addEventListener("click", decreaseTemp, changeColor, changeLandscape);
+    state.currentWeatherButton.addEventListener("click", displayCurrentWeather);
+    state.cityNameInput.addEventListener("input", displayCityName); 
+    state.resetButton.addEventListener("click", setDefaultValues);
+    state.skySelect.addEventListener("change", changeSky);
+};
+
+const loadControls = () => {
+    state.increaseTempButton = document.getElementById("increaseTempControl");
+    state.decreaseTempButton = document.getElementById("decreaseTempControl");
+    state.cityNameInput = document.getElementById("cityNameInput");
+    state.cityNameOutput = document.getElementById("headerCityName");
+    state.currentWeatherButton = document.getElementById("currentTempButton");
+    state.tempValue = document.getElementById("tempValue"); 
+    state.landscape = document.getElementById("landscape");
+    state.resetButton = document.getElementById("cityNameReset");
+    state.skySelect = document.getElementById("skySelect");
+    state.sky = document.getElementById("sky");
+};
+
+const increaseTemp = () => {
+    state.tempDeg += 1;
+    tempValue.textContent = `${state.tempDeg}`
+    changeColor();
+    changeLandscape();
+};
+
+const decreaseTemp = () => {
+    state.tempDeg -= 1;
+    tempValue.textContent = `${state.tempDeg}`;
+    changeColor();
+    changeLandscape();
+};
```

Comment:
Notice that these two methods do essentially the same thing but the difference is -= 1 or += 1 (also in JS you can use ++ or -- which is short hand for doing the same thing).

How could you combine the logic of these two methods into one event handler called handleTempChange()?

What if you pass in the event object as an argument and you check if the event's target has an id `increaseTempControl` or `decreaseTempControl` and depending on the id then you could ++ or --

```javascript
const handleTempChange = (event) => {
    if (event.target.id === 'increaseTempControl') {
        state.tempDeg++;
    } else {
        state.tempDeg--;
    }
    tempValue.textContent = `${state.tempDeg}`;
    changeColor();
    changeLandscape();
};